src/main/java/Fenix.java

```diff
@@ -0,0 +1,17 @@
+public class Fenix {
+    public static void main(String[] args) {
+        String horizontalLine = "_".repeat(60);
+        String greeting = "Hello I'm Fenix";
+        String request = "What can I do for you ?";
+        String farewell = "Bye. Hope to see you again soon!";
+        System.out.println(horizontalLine);
+        System.out.println(greeting + "\n" + request);
```

Comment:
Should avoid writing everything on a main method, divide into different method for easy manage.
There are no spaces so it is hard to keep up with the workflow.
Try to not using the \n in your print statement, instead use another print statement to show that those two are on different lines.